app/Resources/translations/default/ModulesBlockreassuranceAdmin.xlf

```diff
@@ -274,22 +284,22 @@ Comment: Confirm uninstall</note>
       <trans-unit id="6f048a3dd9fc703eca6c05c7cdfb90ec">
         <source>Url link</source>
         <target>Url link</target>
-        <note>Line: 69</note>
+        <note>Line: 68</note>
       </trans-unit>
       <trans-unit id="cb456215c3333db0551bd0788bc258c7">
         <source>Activated</source>
         <target>Activated</target>
-        <note>Line: 84</note>
+        <note>Line: 83</note>
       </trans-unit>
       <trans-unit id="43c7582c40594c3fe32a6634ef9dcb07">
         <source>Deactivated</source>
         <target>Deactivated</target>
-        <note>Line: 85</note>
+        <note>Line: 84</note>
       </trans-unit>
       <trans-unit id="4aa4ae23dfbd6b0941b3749079107b48">
         <source>We recommend 3 blocks at maximum.</source>
```

Comment:
This sentence sounds very French to me, I don't think we would say "at maximum" in English. I suggest saying " We recommend a maximum of 3 blocks." @LouiseBonnard do you agree?

Reply:
PR has already been merged, @Julievrz! But do not hesitate to take note of the improvements you see to suggest them for 1.7.8. :-)

Reply:
Oops, I didn't see. Ok :)